datatypes/papas/Cluster.h

```diff
@@ -0,0 +1,76 @@
+//
```

Comment:
That's correct. In the final install, we should have things appear in a single include/papas directory,